android/filament-android/src/main/java/com/google/android/filament/Scene.java

```diff
@@ -27,35 +47,78 @@
         mNativeObject = nativeScene;
     }
 
+    /**
+     * @return the {@link Skybox} or <code>null</code> if none is set
+     * @see #setSkybox(Skybox)
+     */
     @Nullable
     public Skybox getSkybox() {
         return mSkybox;
     }
 
+    /**
+     * Sets the {@link Skybox}.
+     *
+     * The {@link Skybox} is drawn last and covers all pixels not touched by geometry.
+     *
+     * @param skybox The {@link Skybox} to use to fill untouched pixels,
+     *               or <code>null</code> to unset the {@link Skybox}.
+     */
     public void setSkybox(@Nullable Skybox skybox) {
         mSkybox = skybox;
         nSetSkybox(getNativeObject(), mSkybox != null ? mSkybox.getNativeObject() : 0);
     }
 
+    /**
+     * @return the {@link IndirectLight} or <code>null</code> if none is set
+     * @see #setIndirectLight(IndirectLight)
+     */
     @Nullable
     public IndirectLight getIndirectLight() {
         return mIndirectLight;
     }
 
+    /**
+     * Sets the {@link IndirectLight} to use when rendering the <code>Scene</code>.
+     *
+     * Currently, a <code>Scene</code> may only have a single {@link IndirectLight}.
+     * This call replaces the current {@link IndirectLight}.
+     *
+     * @param ibl The {@link IndirectLight} to use when rendering the <code>Scene</code>
+     *            or <code>null</code> to unset.
+     */
     public void setIndirectLight(@Nullable IndirectLight ibl) {
         mIndirectLight = ibl;
         nSetIndirectLight(getNativeObject(),
                 mIndirectLight != null ? mIndirectLight.getNativeObject() : 0);
     }
 
+    /**
+     * Adds an {@link Entity} to the <code>Scene</code>.
+     *
+     * @param entity The entity is ignored if it doesn't have a {@link RenderableManager} component
+     *               or {@link LightManager} component.<br>
+     *               A given {@link Entity} object can only be added once to a <code>Scene</code>.
+     */
     public void addEntity(@Entity int entity) {
         nAddEntity(getNativeObject(), entity);
     }
 
+    /**
+     * Adds a list of entities to the <code>Scene</code>.
+     *
+     * @param entities Array containing entities to add to the <code>Scene</code>.
```

Comment:
Nit: in general, I think the descriptions for @params should start lowercase, because they're fragments